<launch> 
  	<arg name="uav_name" default="$(optenv UAV_NAME uav1)"/>

 	<arg name="standalone" default="false"/>

  	<arg unless="$(arg standalone)" name="nodelet" value="load"/>
  	<arg     if="$(arg standalone)" name="nodelet" value="standalone"/>
  	<arg unless="$(arg standalone)" name="nodelet_manager" value="$(arg uav_name)_uvdar_nodelet_manager_1"/>
  	<arg     if="$(arg standalone)" name="nodelet_manager" value=""/>

  <!-- <arg name="standalone" default="false"/> -->
  <group ns="$(arg uav_name)">

  <node pkg="nodelet" type="nodelet" name="$(arg uav_name)_uvdar_nodelet_manager_1" args="manager" output="screen"/>

  	<node pkg="nodelet" type="nodelet" name="UVDAR_BP_Tim" args="load uvdar/UVDAR_BP_Tim $(arg nodelet_manager)" output="screen">
		<param name="uav_name" type="string" value="$(arg uav_name)"/>
		<remap from="~$(arg uav_name)/uvdar_bluefox_left/image_raw" to="/$(arg uav_name)/uvdar_bluefox_left/image_raw"/>
	</node>

	<node pkg="nodelet" type="nodelet" name="ImgSubscriber" args="load uvdar/ImgSubscriber $(arg nodelet_manager)" output="screen">
		<param name="uav_name" type="string" value="$(arg uav_name)"/>
		<remap from="~$(arg uav_name)/uvdar_bluefox_right/image_raw" to="/$(arg uav_name)/uvdar_bluefox_right/image_raw"/>
	</node>

	<!-- <node pkg="nodelet" type="nodelet" name="Hello_World" args="load uvdar/Hello_World $(arg nodelet_manager)" output="screen">  -->
	<!-- </node> -->
	</group>
</launch>
